scheduler/src/cook/kubernetes/api.clj

```diff
@@ -731,7 +736,7 @@
   (if (and mode calculate-effective-image-fn)
     (try
       ((util/lazy-load-var-memo calculate-effective-image-fn) kubernetes-config job-submit-time image)
-      (catch Exception e
+      (catch Throwable e
```

Comment:
This seems weird/bad. Why this? (I saw one of the commit mentioned this is to catch an assert error, which, to me, means the assert should probably be changed or moved elsewhere.)

Reply:
updated